vocabularies/sample-relationship.ttl

```diff
@@ -0,0 +1,47 @@
+@prefix dc: <http://purl.org/dc/elements/1.1/> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix smprl: <http://linked.data.gov.au/def/sample-relationship/> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<http://linked.data.gov.au/def/sample-relationship> a owl:Ontology .
+
+smprl:conceptScheme a skos:ConceptScheme ;
+    dct:created "2020-03-24T9:03:21+10:00"^^xsd:dateTime ;
+    dct:creator <http://linked.data.gov.au/org/gsq> ;
+    dct:modified "2020-03-24T9:03:21+10:00"^^xsd:dateTime ;
+    dct:source "Source information from https://www.w3.org/TR/vocab-ssn/#Sample_Relations modified by the Geological Survey of Queensland." ;
+    skos:definition "Samples are often related to other samples, by sub-sampling, topological relationships (stations along a traverse, pixels within an image, probe spots on a polished section, specimens retrieved within a borehole) or as parts of sample processing chains (crushing, splitting, dissecting, disolving). There are an essentially unlimited set of relationships between samples, so the nature of the relationship has its own class."@en ;
+    skos:hasTopConcept smprl:has-originalsample,
+        smprl:has-sample,
+        smprl:is-sample-of ;
+    skos:prefLabel "Sample Relationship"@en .
+smprl:has-originalsample a skos:Concept ;
+    skos:definition "The link to the original sample that is related to the context sample through a chain of isSampleOf relations"@en ;
+    skos:inScheme smprl:conceptScheme ;
+    skos:notation "hasOriginalSample" ;
+    skos:prefLabel "Has OriginalSample"@en ;
+    skos:topConceptOf smprl:conceptScheme .
+
+smprl:has-sample a skos:Concept ;
+    skos:definition "Relation between aï¿½FeatureOfInterest, Site, or parent Sampleï¿½and theï¿½Sampleï¿½used to represent it."@en ;
+    skos:inScheme smprl:conceptScheme ;
+    skos:notation "hasSample" ;
+    skos:prefLabel "Has Sample"@en ;
+    skos:topConceptOf smprl:conceptScheme .
+
+smprl:is-sample-of a skos:Concept ;
+    skos:definition "Relation from aï¿½Sampleï¿½to theï¿½FeatureOfInterest, Site, or parent Sampleï¿½that it is intended to be representative of."@en ;
+    skos:inScheme smprl:conceptScheme ;
+    skos:notation "isSampleOf" ;
+    skos:prefLabel "Is Sample Of"@en ;
+    skos:topConceptOf smprl:conceptScheme .
+    
+<http://linked.data.gov.au/org/gsq>
+    a sdo:Organization ;

```

Comment:
Turtle validator is giving me "_Undefined prefix "sdo:" on line 45._"

Reply:
I get the same. But it's a copy and paste of a Nick addition for defining the organisation.